plugins/export.js

```diff
@@ -42,9 +42,11 @@ window.PostUtils = {formatText, messageHtmlToComponent};
 window.openInteractiveDialog = openInteractiveDialog;
 window.useNotifyAdmin = useNotifyAdmin;
 window.WebappUtils = {
-    browserHistory,
     modals: {openModal, ModalIdentifiers},
 };
+Object.defineProperty(window.WebappUtils, 'browserHistory', {
```

Comment:
I did a bit of magic here so that plugins can still access `window.WebappUtils.browserHistory` as before